staging/src/k8s.io/controller-manager/app/serve.go

```diff
@@ -67,6 +70,9 @@ func NewBaseHandler(c *componentbaseconfig.DebuggingConfiguration, healthzHandle
 	}
 	configz.InstallHandler(mux)
 	mux.Handle("/metrics", legacyregistry.Handler())
+	if utilfeature.DefaultFeatureGate.Enabled(zpagesfeatures.ComponentStatusz) {
+		statusz.Install(mux, "kube-controller-manager", statusz.NewRegistry())
```

Comment:
This directory provides a basis for all controller managers, not just the `kube-controller-manager`.  The name of the component needs to be wired in, not hard coded to `kube-controller-manager`.

Reply:
Done.